### PR TITLE
Read cluster-id from a file instead of argument

### PR DESCRIPTION
**Pull request by nnDarshan**
Changed ceph_integration service to read cluster_id
from a file /var/lib/tendrl/ceph_cluster_id instead
of passing cluster_id to process as an argument.

tendrl-bug-id:Tendrl/ceph_integration#56
Signed-off-by: nnDarshan <darshan.n.2024@gmail.com>

### CODE REVIEW

**Comment by nnDarshan** [comment]
@nthomas-redhat @shtripat @r0h4n Please review

File: tendrl/ceph_integration/manager/utils.py [hunk 1]
```diff
@@ -4,26 +4,19 @@
 import subprocess
 
 LOG = logging.getLogger(__name__)
-TENDRL_CONTEXT = "/etc/tendrl/ceph_integration/tendrl_context"
-NODE_CONTEXT = "/etc/tendrl/node_agent/node_context"
 FSID = "/etc/tendrl/ceph_integration/fsid"
+TENDRL_CONTEXT = "/var/lib/tendrl/ceph_cluster_id"
```
Comment (shtripat) [inline tendrl/ceph_integration/manager/utils.py]: Are we changing the path of the tendrl_context? and is /var/lib the correct path for this? I feel it should rather be /var/run/tendrl may be.

@Tendrl/tendrl-core comments?

File: tendrl/ceph_integration/manager/manager.py [hunk 2]
```diff
@@ -151,13 +150,13 @@ def main():
...
-    if sys.argv:
-        if len(sys.argv) > 1:
-            if "cluster-id" in sys.argv[1]:
-                cluster_id = sys.argv[2]
-                utils.set_tendrl_context(cluster_id)
 
-    m = CephIntegrationManager(utils.get_tendrl_context())
+    cluster_id = utils.get_tendrl_context()
+    if not cluster_id:
+        LOG.error("Could not find cluster_id")
```
Comment (r0h4n) [inline tendrl/ceph_integration/manager/manager.py:156]: could not find cluster_id where? please add path where you are looking for cluster_id or add some logging to get_tendrl_context()

**Review by r0h4n — CHANGES_REQUESTED** [review]

File: tendrl/ceph_integration/manager/utils.py [hunk 3]
```diff
@@ -5,25 +5,18 @@
...
             return cluster_id
-
-
-def set_tendrl_context(cluster_id):
-    with open(TENDRL_CONTEXT, 'wb+') as f:
-        f.write(cluster_id)
-        LOG.info("Tendrl_context.cluster_id==%s created!" % cluster_id)
-        return cluster_id
+    else:
+        return None
```
Comment (r0h4n) [inline tendrl/ceph_integration/manager/utils.py:19]: Add a log here saying no cluster id found at TENDRL_CONTEXT

**Pull request merged**